.github/workflows/conformance.yml

```diff
@@ -29,6 +30,22 @@ jobs:
       with:
         entrypoint: ${{ github.workspace }}/packages/conformance/bin/run
         xfail: "test_verify_with_trust_root"
+
+  conformance-staging:
+    name: Conformance Test (Staging)
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout source
+      uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
+    - name: Setup node
+      uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+      with:
+        node-version: 18
+        cache: npm
+    - name: Install dependencies
+      run: npm ci
+    - name: Build sigstore-js
+      run: npm run build
     - uses: sigstore/sigstore-conformance@ee4de0e602873beed74cf9e49d5332529fe69bf6 # v0.0.11
```

Comment:
I wonder should we disable staging for now since it's breaking?

Reply:
I thought about that, but I kinda wanna leave it here so I know when its working again.